app/assets/javascripts/tylium/modules/fileupload.js

```diff
@@ -1,33 +1,64 @@
 // jQuery.fileUpload  - handles attachment uploads (gem: jquery-fileupload-rails)
 
-// Bind fileUpload on page load.
-document.addEventListener("turbolinks:load", function() {
-  if ($('[data-behavior~=jquery-upload]').length) {
-    $('[data-behavior~=jquery-upload]').fileupload({
-      dropZone: $('#drop-zone'),
-      destroy: function(e, data) {
-        if (confirm('Are you sure?\n\nProceeding will delete this attachment from the associated node.')) {
-          $.blueimp.fileupload.prototype.options.destroy.call(this, e, data);
-        }
-      },
-      paste: function(e, data) {
-        $.each(data.files, function(index, file) {
-          var filename, newFile;
-          filename = prompt('Please provide a filename for the pasted image', 'screenshot-XX.png') || 'unnamed.png';
-          newFile = new File([file], filename, {
-            type: file.type
-          });
-          data.files[index] = newFile;
+function fileUploadInit() {
+  $('[data-behavior~=jquery-upload]').fileupload({
+     autoUpload: true,
+     dropZone: function(){ return $('[data-behavior~=drop-zone]') },
+     pasteZone: function(){ return $('[data-behavior~=drop-zone]') },
+     singleFileUploads: true,
+     destroy: function(e, data) {
+       if (confirm('Are you sure?\n\nProceeding will delete this attachment from the associated node.')) {
+         $.blueimp.fileupload.prototype.options.destroy.call(this, e, data);
+       }
+     }
```

Comment:
I don't think we need this anymore since we are auto-uploading. This was for the staged area delete button in case users wanted to remove files from the queue.

Reply:
> Proceeding will delete this attachment from the associated node.

Isn't this the delete button to remove the file completely?

Reply:
Verified, this is for the button to delete an existing already uploaded attachment, and should not be removed.

Reply:
This did remind me though. We hadn't removed the Add, Start, and Remove buttons for the staging area. So I've removed those now as well.